rero/rero-ils
# Wep fix delayed bulk indexing

## Why are you opening this PR?

- scheduled indexing task was not using the right indexer class

## How to test?

- `pipenv run invenio utils reindex -t doc` and start task `rero_ils.modules.task.process_bulk_queue` from console. Verify the documents were indext correctly.
- repeat with `process_bulk_queue.delay()`. A beat worker must be running for this.
- See if the scheduled task `indexer` runs correctly 

## Code review check list
